view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,272 @@
+<script>
```

Comment:
please move the js functions to https://github.com/friendica/friendica/blob/develop/view/theme/frio/js/mod_admin.js and include it with ```<script type="text/javascript" src="view/theme/frio/js/mod_admin.js"></script>```